src/explore-education-statistics-admin/src/pages/release/pre-release/components/PreReleaseUserAccessForm.tsx

```diff
@@ -141,8 +137,9 @@ const PreReleaseUserAccessForm = ({
                 test(value?: string) {
                   if (value) {
                     const emails = splitAndTrimLines(value);
+                    const schema = Yup.string().email();
                     const indexOfFirstInvalid = emails.findIndex(
-                      email => !isValidEmail(email),
+                      email => !schema.isValidSync(email),
```

Comment:
Just checking, but assuming the previous validation was more restrictive on purpose, are we now making this too lax?

Have we checked this with Cam?

Reply:
I've checked with Cam, we don't think the restriction was on purpose and he's fine with it using the Yup validation instead. This is the ticket https://dfedigital.atlassian.net/browse/EES-3582 the regex was added for.